spring-kafka/src/main/java/org/springframework/kafka/listener/ContainerPausingBackOffHandler.java

```diff
@@ -51,4 +53,9 @@ public void onNextBackOff(@Nullable MessageListenerContainer container, Exceptio
 		}
 	}
 
+	@Override
+	public void onNextBackOff(MessageListenerContainer container, TopicPartition partition, long nextBackOff) {
```

Comment:
Given the interface states `container` is `@Nullable`, maybe a null check is in order here?